src/Interop/LibLLVM/CMakeLists.txt

```diff
@@ -0,0 +1,208 @@
+cmake_minimum_required(VERSION 3.0.0)
```

Comment:
This is missing handling of rthe RC file, which as discussed in prior PRs is used to mark the version information in the file. A lot of effort went into ensuring that was done correctly and consistently for the build. Please restore that functionality.